addons/website_project_issue/controllers/main.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import werkzeug
+
+from openerp import http
+from openerp.http import request
+
+from openerp.addons.website_portal.controllers.main import website_account
+
+class WebsiteAccount(website_account):
+    @http.route(['/my', '/my/home'], type='http', auth="user", website=True)
+    def account(self):
+        response = super(WebsiteAccount, self).account()
+        project_issues = request.env['project.issue'].search([])
+        response.qcontext.update({
+            'issues': project_issues
+        })
+        return response
+
+class website_project_issues(http.Controller):
+    @http.route(['/my/issues/<model("project.issue"):issue>'], type='http', auth="user", website=True)
+    def issues_followup(self, issue=None):
+        has_timesheet_enabled = issue._fields.get('timesheet_ids')
+        return request.website.render("website_project_issue.issues_followup", {
+            'issue': issue,
+            'timesheets': [(timesheet.date, timesheet.user_id.name, timesheet.name) for timesheet in issue.sudo().timesheet_ids] if has_timesheet_enabled else False
+        })
+
+    @http.route(['/issue/<model("project.issue"):issue>/post'], type='http', auth="user", website=True)
+    def post(self, issue=None, **post):
+        message = post.get('comment')
```

Comment:
@rim-odoo  Template already had textarea with required attribute but safari browser won't support that so now added a condition.